src/components/ProductOrder/ProductOrder.js

```diff
@@ -168,10 +196,13 @@ export default class ProductOrder extends Component {
 
 ProductOrder.propTypes = {
   addToCart: PropTypes.func,
-  className: PropTypes.string
+  className: PropTypes.string,
+  currencyKey: PropTypes.number.isRequired,
+  symbol: PropTypes.string
```

Comment:
Aha, so it is a string! :) Please change the symbol propTypes in other files to string too :)